platform/core/src/services/UICustomizationService/types.ts

```diff
@@ -0,0 +1,33 @@
+import Command from '../../types/Command';
+
+export interface BaseUICustomization extends Record<string, unknown> {
+  id: string;
+  customizationType?: string;
+  description?: string;
+  label?: string;
+  commands?: Command[];
+}
+
+export interface LabelUICustomization extends BaseUICustomization {
+  label: string;
+}
+
+export interface CodeUICustomization extends BaseUICustomization {
+  code: string;
+}
+
+export interface CommandUICustomization extends BaseUICustomization {
+  commands: Command[];
+}
+
+export type UICustomization =
+  | BaseUICustomization
+  | LabelUICustomization
+  | CommandUICustomization
+  | CodeUICustomization;
+
+export default UICustomization;
+
+export interface UIConfiguration {
```

Comment:
I think this is the one that need to be imported for the constructor?

Reply:
I removed this one - it was old/not well setup.  Probably it should eventually get added to the constructor as a properly typed item.